pkg/application/app.go

```diff
@@ -141,7 +159,7 @@ func (app *Avalanche) CreateSidecar(sc *models.Sidecar) error {
 	// but better safe than sorry
 	exists, err := app.ChainIDExists(sc.ChainID)
 	if err != nil {
-		return err
+		return fmt.Errorf("unable to determine if chainID is unique: %s", err)
```

Comment:
it is best practice to use %w to wrap errors, instead of %s, when using Errorf

Reply:
good catch, fixed